lib/levels.js

```diff
@@ -119,7 +113,36 @@ function getLevel (level) {
 function isLevelEnabled (logLevel) {
   const { values } = this.levels
   const logLevelVal = values[logLevel]
-  return logLevelVal !== undefined && (logLevelVal >= this[levelValSym])
+  return logLevelVal !== undefined && this[levelCompSym](logLevelVal, this[levelValSym])
+}
+
+/**
+ *
+ * @description compare `current` level with `expected` to make sure `current` level is enabled or not
+ * @param {SORTING_ORDER} direction
+ * @param {number} current
+ * @param {number} expected
+ * @returns {boolean}
+ */
+function compareLevel (direction, current, expected) {
```

Comment:
Would this be a clearer signature?
```suggestion
function isLevelEnabled (levelValue, threshold, sortOrder = SORTING_ORDER.DESC) {
```

Reply:
The signature of this function starts from order to be able to create a single sorting function using `bind` with predefined first argument `direction`.